index.html

```diff
@@ -0,0 +1,187 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Consul</title>
+    <link rel="stylesheet" href="style.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+  </head>
+  <body>
+    <header>
+      <div class="logo">
+        <a href="#"><img src="./img/consul-logo.png" alt="" /></a>
+      </div>
+      <nav class="navbar">
+        <div class="hamburger">
+          <img src="./img/Hamburger_icon.svg.png" alt="Menu Hamburger" />
+        </div>
+        <ul class="nav-links">
+          <li class="close-menu-btn">
+            <button type="submit">
+              <span>&times;</span>
+            </button>
+          </li>
+          <li><a href="index.html" id="current">Home</a></li>
+          <li><a href="about.html">About</a></li>
+        </ul>
+      </nav>
+    </header>
```

Comment:
- [ ] On Desktop screens, the navbar is:
- Divided into a top dark bar and a bottom white bar.
- The top short bar section has social media links, logout, and language links aligned to the right.
- The taller bottom bar has a logo and a list of page links. Kindly add the remaining page links. 
Remember you have to follow the provided design guidelines for this project.
- Use the required background colour for the navigation bar. This helps with the colour contrast especially when scrolling down the page, the navbar obstructs and overlaps other elements in the page. 

_See image below._

![image](https://user-images.githubusercontent.com/49621279/123976067-b4ef1a80-d972-11eb-9fa3-a334f834651f.png)



- [ ] Mobile Screen
- Good job implementing the drop-down menu, it works perfectly well 👍🏾 
Change the background image of the menu to fit in with the provided design template.
_See image below_
![image](https://user-images.githubusercontent.com/49621279/123953294-ed372e80-d95b-11eb-9507-9af4f32bf3c5.png)